changelog/v1.26.4-patch2/envoy-fork.yaml

```diff
@@ -0,0 +1,9 @@
+changelog:
+  - type: DEPENDENCY_BUMP
+    dependencyRepo: envoy-fork
+    dependencyOwner: solo-io
+    dependencyTag: v1.26.4
+    description: >
+      Migrate to fork that is based off of 1.26.4. It includes long term backports.
+      These backports are not fully merged upstream but make external processing actually function as needed.
```

Comment:
Do we intend to switch back to upstream once these changes are merged?

Reply:
nope never as that will be merged into 1.27 which we are not on in this branch